tests/FamilyHubs.ReferralApi.UnitTests/WhenUsingReferralCommands.cs

```diff
@@ -161,6 +270,8 @@ public static ReferralDto GetReferralDto()
                 "Robert.Brown@yahoo.co.uk",
                 "0131 222 3333",
                 "",
+                DateTime.UtcNow,
```

Comment:
I am not sure about this so no action is needed but do we need to do any testing about this date? I can see new bit of logic has been added around comparing text data field

Reply:
Yes that is so it displays things in the correct order